File: status.c
#include <stdio.h>

int main() {
    char name[50];
    int units;

    printf("Enter student name: ");
    scanf("%s", name);

    printf("Enter number of registered units: ");
    scanf("%d", &units);

    char* status = (units > 7) ? "Overload - Approval Required" : "Registration Accepted";

    printf("\n--- Final Summary ---\n");
    printf("Student Name: %s\n", name);
    printf("Units: %d\n", units);
    printf("Status: %s\n", status);

    return 0;
}

File: status.java
import java.util.Scanner;

public class RegistrationChecker {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);

        System.out.print("Enter student name: ");
        String name = input.nextLine();

        System.out.print("Enter number of registered units: ");
        int units = input.nextInt();

        String status = (units > 7) ? "Overload - Approval Required" : "Registration Accepted";

        System.out.println("\n--- Final Summary ---");
        System.out.println("Student Name: " + name);
        System.out.println("Units: " + units);
        System.out.println("Status: " + status);
    }
}
